utils/swift_build_support/swift_build_support/shell.py

```diff
@@ -80,40 +88,74 @@ def call(command, stderr=None, env=None, dry_run=None):
             "': " + e.strerror)
 
 
+def capture(command, stderr=None, env=None, dry_run=None, print_command=True):
```

Comment:
+1 for `echo`.  (`@echo off` anyone?)

And I also agree that even though `capture` would not usually be echoed, consistency is important.